README.md

```diff
@@ -22,21 +23,31 @@ jobs:
           circleci-jobs: build_doc
           job-title: Check the rendered docs here!
       - name: Check the URL
+        if: github.event.status != 'pending'
         run: |
           curl --fail ${{ steps.step1.outputs.url }} | grep $GITHUB_SHA
 
 ```
-
+- The `if: "${{ github.event.context == 'ci/circleci: build_doc' }}"`
+  conditional in the `job` is helpful to limit the number of redirector
+  actions that your repo will run to avoid polluting your GitHub actions
+  logs. The `circleci-jobs` (below) should be labeled correspondingly.
 - The `artifact-path` should point to an artifact path from your CircleCI
   build. This is typically whatever follows the CircleCI artifact root path,
   for example `0/test_artifacts/root_artifact.md`.
 - The `circleci-jobs` is a comma-separated list of jobs to monitor, but usually
   there is a single one that you want an artifact path for.
   The default is `"build_docs,build,doc"`, which will look for any
-  jobs with these names and create an artifacts link for them.
+  jobs with these names and create an artifacts link for them. If you have
+  multiple jobs to consider, make sure you adjust your `if:` statement (above)
+  correspondingly.
 - The `job-title` corresponds to the name of the action job as it will appear
   on github. It is **not** the circle-ci job you want the artifacts for
   (this is `circleci-jobs`). This field is optional.
+- The action has an outtput ``url`` that you can use in downstream steps, but
+  this URL will only point to a valid artifact once the job is complete, i.e.,
+  `github.event.status` is either `'success'`, `'fail'`, or (maybe) `'error'`,
+  not `'pending'`.
```

Comment:
@asmeurer this bit, implemented this way for this repo

https://github.com/larsoner/circleci-artifacts-redirector-action/blob/ecc5ada864e38a28a3a1a28588d39459793bccfa/.github/workflows/status.yml#L38

I think things have been green since then...

https://github.com/larsoner/circleci-artifacts-redirector-action/actions